index.d.ts

```diff
@@ -0,0 +1,17 @@
+declare class Hasher {
+    constructor(size?: 224 | 256 | 384 | 512);
+
+    update(data: Buffer | string): this;
```

Comment:
The `update` function, not supported `encoding` arguments

https://github.com/phusion/node-sha3/blob/master/src/index.js#L11-L22
The `update` function, This modification can to be consistent with the documentation
```diff
diff --git a/src/index.js b/src/index.js
index 477ed82..8f1eb57 100644
--- a/src/index.js
+++ b/src/index.js
@@ -8,14 +8,14 @@ const createHash = ({ padding }) => function Hash(size = 512) {
 
   const sponge = new Sponge({ capacity: size, padding });
 
-  this.update = (input) => {
+  this.update = (input, encoding = 'utf8') => {
     if (Buffer.isBuffer(input)) {
       sponge.absorb(input);
       return this;
     }
 
     if (typeof input === 'string') {
-      return this.update(Buffer.from(input, 'utf8'));
+      return this.update(Buffer.from(input, encoding));
     }
 
     throw new TypeError('Not a string or buffer');
```

see https://github.com/phusion/node-sha3#updatedata-encodingutf8